usaspending_api/etl/tests/integration/test_load_to_from_delta.py

```diff
@@ -682,6 +701,68 @@ def verify_delta_table_loaded_to_delta(
     assert equal_datasets(dummy_data, received_data, TABLE_SPEC[delta_table_name]["custom_schema"], ignore_fields)
 
 
+def verify_delta_query_loaded_to_delta(
```

Comment:
Maybe there's a weird merge history. This isn't all new here is it? Doesn't this python module already have all this code. Why would it be newly added in this PR.

Reply:
Or was it just moved around in the module?

Reply:
Was not in the latest PR.